# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/webrtc-audio-processing-1/hotfix-cstdint.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- webrtc-audio-processing-v2.1/webrtc/modules/audio_processing/aec3/multi_channel_content_detector.h.vanilla	2025-07-16 15:24:33.147984316 +0200
+++ webrtc-audio-processing-v2.1/webrtc/modules/audio_processing/aec3/multi_channel_content_detector.h	2025-07-16 15:24:39.419191535 +0200
@@ -16,6 +16,7 @@
 #include <memory>
 #include <optional>
 #include <vector>
+#include <cstdint>
 
 namespace webrtc {
 
--- webrtc-audio-processing-v2.1/webrtc/rtc_base/trace_event.h.vanilla	2025-07-16 15:23:56.717958099 +0200
+++ webrtc-audio-processing-v2.1/webrtc/rtc_base/trace_event.h	2025-07-16 15:24:14.312837862 +0200
@@ -26,6 +26,8 @@
 #endif
 // IWYU pragma: end_exports
 
+#include <cstdint>
+
 #if !defined(RTC_USE_PERFETTO)
 #include <string>
 
